docs/v3.x/concepts/models.md

```diff
@@ -215,7 +215,10 @@ To improve the Developer Experience when developing or using the administration
 
 ### Exceptions
 
-- `uid` — This field type allows a `targetField` key. The value is the name of an attribute thas has `string` of `text` type.
+**`uid`**
```

Comment:
Hello!

You list doesn't looks like the same as the others in this page.

Can you please use the same format like that:

- `targetField ` (text) - The value is the na...

Thank you.